inc/sf_c_util/fabric_string_list_result.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef FABRIC_STRING_LIST_RESULT_H
+#define FABRIC_STRING_LIST_RESULT_H
+
+#ifdef __cplusplus
+#include <cstddef>
+#else
+#include <stddef.h>
+#endif
+
+#include "windows.h"
+
+#include "umock_c/umock_c_prod.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+    typedef struct FABRIC_STRING_LIST_RESULT_TAG* FABRIC_STRING_LIST_RESULT_HANDLE;
+
+    MOCKABLE_FUNCTION(, FABRIC_STRING_LIST_RESULT_HANDLE, fabric_string_list_result_create, ULONG, nStrings, const wchar_t**, strings);
```

Comment:
>wchar_t [](http://example.com/codeflow?start=115&length=7)

Likely needs wchar include?

Reply:
no, it doesn't because it gets it from stddef.h.

I know that because I had the exact same comment on one of Parth's CR and he educated me. 

```

7.19 Common definitions <stddef.h>
1 The header <stddef.h> defines the following macros and declares the following types
...
wchar_t
which is an integer type whose ran...
```